packages/database/src/realtime/TransportManager.ts

```diff
@@ -32,10 +32,16 @@ import { WebSocketConnection } from './WebSocketConnection';
 export class TransportManager {
   private transports_: TransportConstructor[];
 
+  static globalTransportInitialized_ = false;
+
   static get ALL_TRANSPORTS() {
     return [BrowserPollConnection, WebSocketConnection];
   }
 
+  static get IS_TRANSPORT_INITIALIZED() {
```

Comment:
Please add comments explaining the meaning of this function (and the associated variable).